bats/fb-test-foreman-ansible.bats

```diff
@@ -19,13 +19,15 @@ setup() {
   echo "- command: uptime" > "/etc/ansible/roles/${ROLE_NAME}/tasks/main.yml"
 
   # check if smart_proxy_ansible finds it and can import it
-  hammer ansible roles fetch --proxy-id 1 | grep "${ROLE_NAME}"
+  if ! hammer --output csv --no-headers ansible roles list --search="name=${ROLE_NAME}" | grep "${ROLE_NAME}"; then
+    hammer ansible roles fetch --proxy-id 1 | grep "${ROLE_NAME}"
+  fi
   hammer ansible roles sync --proxy-id 1 --role-names "${ROLE_NAME}"
 
   # sync creates a task, so let's wait for it to complete
   tWaitForTask SyncRolesAndVariables
 
-  hammer ansible roles list | grep "${ROLE_NAME}"
+  hammer --output csv --no-headers ansible roles list --search="name=${ROLE_NAME}" | grep "${ROLE_NAME}"
```

Comment:
Not for this PR, but this feels like one of those "our bats is too old so we can't use `assert_output` cases". Should we modify https://github.com/theforeman/forklift/pull/1634 to also pull in `bats-assert`?

Reply:
You mean https://github.com/bats-core/bats-assert? Yeah, certainly something we could try pulling in.